custom_platforms.js

```diff
@@ -182,6 +185,7 @@ module.exports = {
         "path": "extension/safari",
         "configuration": {
             "removeRulePatterns": [
+                "\\$(.*,)?remove(param|header)",
```

Comment:
it would be great to have here examples of rules for regexp, isn't it?
so it could be easier to fix issues if any occur

for example:
```
"removeRulePatterns": [
    // ||example.org^$removeheader=location
    // ||example.org^$xmlhttprequest,removeparam=param
    // ||example.org^$xmlhttprequest,removeheader=location
    "\\$(.*,)?remove(param|header)",
```

Reply:
I'm not sure if it's necessary to do that, the regex looks fine.
I've updated the description with a link to examples from `regex101.com`.